nginx_config_reloader/copy_files.py

```diff
@@ -0,0 +1,24 @@
+import logging
+from itertools import chain
+from subprocess import check_output, STDOUT
+
+from nginx_config_reloader.settings import SYNC_IGNORE_FILES
+
+logger = logging.getLogger(__name__)
+
+
+def safe_copy_files(src, dest):
+    cmd = [
+        # Adding a / at the end copies contents of the dir and not the dir itself
+        'rsync', src + '/', dest,
+        # Delete and archive without copying over permissions, aka -da but without -p (-a equals -rlptgoD)
+        '-drltgoD',
+        '--chown', 'root:root',
+        '--copy-links',  # Follow symlinks and copy actual file
+        # Dirs default to 0755 to read. Remove setuid bits. Remove executability for others
+        '--chmod="D755,-s,Fo-wx"',
+    ]
+    cmd.extend(["--exclude=\"{}\"".format(pattern) for pattern in SYNC_IGNORE_FILES])
```

Comment:
In other places, we use `--exclude` together with `--filter` (like https://github.com/ByteInternet/hypernode-kamikaze3/blob/84d92b1cf438bc2f640ce8e465b2a887dfb65dc3/kamikaze3/importer/file_transfer.py#L211). Not sure what that adds, maybe something to check out?

Reply:
I'm not sure that's necessary at this point, because `--exclude` seems to do the trick